app/src/main/java/org/wikipedia/feed/onthisday/OnThisDayActivity.java

```diff
@@ -13,21 +13,17 @@
 
 public class OnThisDayActivity extends SingleFragmentActivity<OnThisDayFragment> {
     public static final String AGE = "age";
+    public static final String YEAR = "year";
     public static final String WIKISITE = "wikisite";
 
-    public static Intent newIntent(@NonNull Context context, int age, WikiSite wikiSite, InvokeSource invokeSource) {
+    public static Intent newIntent(@NonNull Context context, int age, @NonNull WikiSite wikiSite, @NonNull InvokeSource invokeSource, int year) {
```

Comment:
Can we move the `int year` to the left of `InvokeSource invokeSource`? I'd like to see the `invokeSource` stay on the last position.